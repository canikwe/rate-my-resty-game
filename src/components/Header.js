import React, { Component } from 'react'

class Header extends Component {
  render() {
    return(
      <h1>Welcome to Rate My Resty!</h1>
    )
  }
}

export default Header