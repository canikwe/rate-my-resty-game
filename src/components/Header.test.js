import React from 'react'
import Header from './Header'
import renderer from 'react-test-renderer'

describe('Header', () => {
  it('Renders on the Header directly to the page', () => {
    const tree = renderer
    .create(<Header />)
    .toJSON()
    expect(tree).toMatchSnapshot()
  })
})