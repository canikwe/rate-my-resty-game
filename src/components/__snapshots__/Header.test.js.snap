// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Renders on the Header directly to the page 1`] = `
<h1>
  Welcome to Rate My Resty!
</h1>
`;
